x-pack/plugins/apm/server/lib/service_group_query.ts

```diff
@@ -6,17 +6,11 @@
  */
 
 import { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
-import { SERVICE_NAME } from '../elasticsearch_fieldnames';
-import { ServiceGroup } from '../service_groups';
+import { kqlQuery } from '@kbn/observability-plugin/server';
+import { ServiceGroup } from '../../common/service_groups';
 
 export function serviceGroupQuery(
   serviceGroup?: ServiceGroup | null
 ): QueryDslQueryContainer[] {
-  if (!serviceGroup) {
-    return [];
-  }
-
-  return serviceGroup?.serviceNames
-    ? [{ terms: { [SERVICE_NAME]: serviceGroup.serviceNames } }]
-    : [];
+  return serviceGroup ? kqlQuery(serviceGroup?.kuery) : [];
```

Comment:
I don't know about this file, this doesn't do that much for us to have a function to it. But it's up to up.

Reply:
Agree. Better to inline at this point.

Reply:
Actually, it reads quite nicely atm:
```ts
 ...serviceGroupQuery(serviceGroup),
```